.gitignore

```diff
@@ -0,0 +1,4 @@
+# composer
+vendor
```

Comment:
```suggestion
```

Reply:
For now, let's include the vendor folder in the git repository.

This way, when folks git clone/etc the plugin just works instead of requiring the composer install always.

I feel we won't be updating these libraries that often and so it's useful to keep the vendor folder in there and make it easier for folks to work with the repo.